test/Kestrel.Core.Tests/IntegerDecoderTests.cs

```diff
@@ -34,6 +34,21 @@ public void IntegerDecode(int i, int prefixLength, byte[] octets)
             Assert.Equal(i, decoder.Value);
         }
```

Comment:
Can you add a round trip test? for 0 to max int, encode, decode, and verify?

Reply:
Turns out this might not be feasible. {0 .. int.maxint}X{1 .. 8 prefix length} by my estimate is going to take 2.5 hours.

Reply:
You should still do the round trip test with IntegerData. Maybe add a few more cases if you can think of anything interesting, but I agree every possible input seems like a bit much.

Reply:
I ended up opting for {0 .. 1 << 8} U {int.MaxValue - 1 << 8 .. int.MaxValue} I think that's a reasonable range.